deploy/environments/applications.yaml

```diff
@@ -57,6 +58,7 @@ spack:
         - 'py-neurodamus+all_deps'
         - py-nexusforge
         - py-nexus-sdk
+        - py-randomaccessbuffer
```

Comment:
This seems like a support library. Does this need to be a end-user facing module?

Reply:
Indeed, I'll remove it from the list.